infrastructure/api_gateway.tf

```diff
@@ -0,0 +1,58 @@
+resource "aws_api_gateway_rest_api" "get-objects-API" {
+  name        = "get-objects-API"
+  description = "for AWS Lambda trigger"
+}
+
+resource "aws_api_gateway_resource" "proxy" {
+  rest_api_id = "${aws_api_gateway_rest_api.get-objects-API.id}"
+  parent_id   = "${aws_api_gateway_rest_api.get-objects-API.root_resource_id}"
+  path_part   = "{proxy+}"
+}
```

Comment:
do you need the proxy? 
I think you can remove it

Reply:
true